doc/source/whatsnew/v0.23.2.txt

```diff
@@ -66,7 +66,7 @@ Bug Fixes
 
 **Reshaping**
 
--
+- Bug in: DataFrame.pivot() function where error was raised when multiple columns are set as index
```

Comment:
Can you move this to the `0.24.0.txt` file? And you can mention it in the Enhancements section

(as this is an new feature, not really a bug fix, I think, as it was just not yet implemented)